server/handlers/handlers_test.go

```diff
@@ -260,7 +260,7 @@ func TestHandleCreatedActivity(t *testing.T) {
 			setupAPI: func(mockAPI *plugintest.API) {
 				mockAPI.On("GetDirectChannel", "mockUserID-1", "mockUserID-2").Return(&model.Channel{Id: testutils.GetChannelID()}, nil).Times(1)
 				mockAPI.On("GetUser", testutils.GetUserID()).Return(testutils.GetUser(model.ChannelAdminRoleId, "test@test.com"), nil).Once()
-				mockAPI.On("LogDebug", "Post generated").Times(1)
+				mockAPI.On("LogDebug", "Post generated", "PostID", mock.AnythingOfType("string")).Times(1)
```

Comment:
Why are you using mock.AnythingOfType? You should use this only when you can't use the ID here.